redi-connect-backend/lib/email/email.js

```diff
@@ -269,12 +269,13 @@ const sendVerificationEmail = ({
   verificationToken,
   rediLocation
 }) => {
+  const signupType = userType
```

Comment:
This gets muddled. Let's clear it up by:
1. use `userType: signupType` on line 268.
2. on line 273, write `const userType` 
3. on line 276, use `signupType`

Reply:
Nice, I like this clean solution much better. Thanks, Eric!